main.nf

```diff
@@ -411,19 +345,19 @@ process countstat2 {
     publishDir "${params.outdir}/countstat", mode: 'copy'
 
     input:
-    file ('*.bed') from bed_dedup.toSortedList()
+    file input from bed_total .mix(bed_dedup) .toSortedList()
 
     output:
-    file 'Count_stat_bed_dedup_reads' into countstat2_results
+    file 'Reads_count_statistics.txt' into countstat_results
```

Comment:
* I prefer not to have sentence case capitalisation results files (typically lower case)
* _Reads_ should be _Read_ (singular). Read count statistics.